### PR TITLE
Environment config updates

### PR DESCRIPTION
**Pull request by tboerger**
* Raised crowbar version
* Added initial rainbows worker

### CODE REVIEW

File: crowbar_framework/config/crowbar.env [hunk 1]
```diff
@@ -1,4 +1,6 @@
-export CROWBAR_VERSION=1.10
+export RAINBOWS_WORKERS=5
+
+export CROWBAR_VERSION=3.0
```
Comment (dirkmueller) [inline crowbar_framework/config/crowbar.env]: maybe 2.95 until we released 3.0 ?

Comment (tboerger) [inline crowbar_framework/config/crowbar.env]: But we are working on 3.0, this is the upcoming version :)

**Comment by rsalevsky** [comment]
+1

File: crowbar_framework/config/crowbar.env [hunk 2]
```diff
@@ -1,4 +1,6 @@
-export CROWBAR_VERSION=1.10
+export RAINBOWS_WORKERS=5
```
Comment (MaximilianMeister) [inline crowbar_framework/config/crowbar.env]: @tboerger does the `RAINBOWS_WORKERS` env influence only development or also production?

Comment (tboerger) [inline crowbar_framework/config/crowbar.env]: Just development as production uses a custom rainbows config.

**Comment by MaximilianMeister** [comment]
> Just development as production uses a custom rainbows config.

okay thanks.

:+1: then

**Pull request merged**